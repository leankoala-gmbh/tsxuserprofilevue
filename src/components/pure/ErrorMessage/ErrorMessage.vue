<template>
  <div class="errorMessage text-xs text-red-500">
    <slot />
  </div>
</template>
