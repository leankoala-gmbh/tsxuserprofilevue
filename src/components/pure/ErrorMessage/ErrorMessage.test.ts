import { mount } from '@vue/test-utils'
import { describe, expect, test } from 'vitest'
import ErrorMessage from './ErrorMessage.vue'

describe('ErrorMessage.vue', () => {
  test('check snapshpt', () => {
    const wrapper = mount(ErrorMessage, {
      slots: {
        default: 'Error message'
      }
    })
    expect(wrapper.html()).toMatchSnapshot()
  })
})
