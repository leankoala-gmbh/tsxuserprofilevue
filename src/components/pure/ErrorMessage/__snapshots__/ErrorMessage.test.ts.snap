// Vitest Snapshot v1

exports[`ErrorMessage.vue > check snapshpt 1`] = `"<div class=\\"errorMessage text-xs text-red-500\\">Error message</div>"`;
